### ISSUE TITLE
HA guide with docker is subpar

### ISSUE THREAD

**Issue by robbyoconnor** [RocketChat/docs-old#980]
This is more for me to have an issue to track it -- but the [guide ](https://rocket.chat/docs/installation/docker-containers/high-availability-install/) isn't great -- and considering most people will likely copy/paste -- it should contain code ready for that use. I'll take on the work to fix this.

**Comment by Sagar2366** [comment]
yes, you are right, only thing is not every one is trying to copy paste directly. But it would be great if all the instructions are given properly and stepwise, they should be in layman words that anyone with could understand

**Comment by robbyoconnor** [comment]
@Sagar2366  The issue is that a layman should not be setting this up, a professional who is knowledgeable should be.  It is assumed that people know what they're doing, or they should hire someone who does. That wasn't my rationale for opening this. I saw several areas where this information was rather outdated.

**Comment by robbyoconnor** [comment]
It appears I dropped the ball

**Comment by Rodriq** [comment]
High availability in docs https://docs.rocket.chat/quick-start/deploying-rocket.chat/rapid-deployment-methods/docker-and-docker-compose/docker-containers/high-availability-install
